src/main/java/core/selenium/WebDriverConfig.java

```diff
@@ -40,6 +45,7 @@ private Properties readConfigurationFile() {
 
     /**
      * this method ensure that only one instance is created according to the build pattern.
```

Comment:
Please change the first letter to UpperCase T.

Reply:
Solved